README.md

```diff
@@ -76,8 +76,28 @@ Feeding the data, url for download. Ultimately, the links have been sent to your
 
 ![](P2.gif?raw=true)
 
-### With Docker
-Run the following commands
+### With Docker (Use any of the two methods)
+#### Pull Docker image from Docker Hub and start the container
+Make sure you are logged into Docker Hub from your command line by running `docker login`
```

Comment:
Can you please add instructions to install docker or a link where a user can find steps to install it?